hack/community/generate.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+# Generate the new community manifests, given the current wmco manifests.
+
+# Must be ran within wmco root.
+
+# Example:
+# Update community manifests v5.0.1
+# Run: bash ./hack/community/generate.sh
+
+# Install tools
+echo "Install yq"
+curl -L https://github.com/mikefarah/yq/releases/download/v4.13.5/yq_linux_amd64 -o /tmp/yq && chmod +x /tmp/yq
+echo "yq installed"
+
+OPERATOR_MANIFESTS="bundle/manifests"
+OPERATOR_METADATA="bundle/metadata"
+CSV="${OPERATOR_MANIFESTS}/windows-machine-config-operator.clusterserviceversion.yaml"
+FILE=../community-operators-prod/operators/community-windows-machine-config-operator/${OPERATOR_VERSION}
+echo "Extracting WMCO version from CSV"
+OPERATOR_VERSION=$(/tmp/yq eval '.spec.version' "${CSV}")
+echo "Current WMCO version: ${OPERATOR_VERSION}"
+
+CO_PROJECT="community-operators-prod"
+CO_REPO="redhat-openshift-ecosystem/${CO_PROJECT}"
+CO_GIT="https://github.com/${CO_REPO}.git"
+CO_FORK="https://${GITHUB_USER}:${GITHUB_TOKEN}@github.com/${GITHUB_USER}/${CO_PROJECT}.git"
+CO_DIR=$(mktemp -d)
+CO_OPERATOR_DIR="${CO_DIR}/operators/windows-machine-config-operator"
```

Comment:
The name of the directory for [community](https://github.com/redhat-openshift-ecosystem/community-operators-prod/tree/main/operators/community-windows-machine-config-operator) is `community-windows-machine-config-operator`.